src/browser/components/icons/Icons.tsx

```diff
@@ -326,7 +326,7 @@ export const PinIcon = (): JSX.Element => (
   <IconContainer icon={pin} width={12} />
 )
 export const SaveFavorite = (): JSX.Element => (
```

Comment:
Missing the `Icon` suffix here, not sure if it got missed anywhere else, but would be nice to have them all be consistent! :-)

Reply:
Thanks! Went through and found a couple more misnamed icons